src/OrchardCore.Modules/OrchardCore.Markdown/Handlers/MarkdownBodyPartHandler.cs

```diff
@@ -13,7 +14,9 @@ public class MarkdownBodyPartHandler : ContentPartHandler<MarkdownBodyPart>
     {
         private readonly ILiquidTemplateManager _liquidTemplateManager;
         private readonly HtmlEncoder _htmlEncoder;
-        private HtmlString _bodyAspect;
```

Comment:
why change from `HtmlString`?

Reply:
Unable to cast `IHtmlContent` to `HtmlString`

Reply:
Please change this back to `HtmlString`. When you fix where it is it being cached it will not require changing